package/kedro_viz/server.py

```diff
@@ -677,32 +663,25 @@ def _call_viz(
 
         _DATA = _load_from_file(load_file)
     else:
-        if KEDRO_VERSION.match(">=0.15.0"):
-            # pylint: disable=import-outside-toplevel
-            if KEDRO_VERSION.match(">=0.16.0"):
-                from kedro.framework.context import KedroContextError
-            else:
-                from kedro.context import (  # pylint: disable=no-name-in-module,import-error
-                    KedroContextError,
-                )
-
-            try:
-                if project_path is not None:
-                    context = get_project_context(
-                        "context", project_path=project_path, env=env
-                    )
-                else:
-                    context = get_project_context("context", env=env)
-                pipelines = _get_pipelines_from_context(context, pipeline_name)
-            except KedroContextError:
-                raise KedroCliError(ERROR_PROJECT_ROOT)
-            _CATALOG = context.catalog
-
+        # pylint: disable=import-outside-toplevel
+        if KEDRO_VERSION.match(">=0.16.0"):
+            from kedro.framework.context import KedroContextError
         else:
-            # Kedro 0.14.*
-            if pipeline_name:
-                raise KedroCliError(ERROR_PIPELINE_FLAG_NOT_SUPPORTED)
-            pipelines, _CATALOG = _get_pipeline_catalog_from_kedro14(env)
+            from kedro.context import (  # pylint: disable=no-name-in-module,import-error
+                KedroContextError,
+            )
+
+        try:
+            if project_path is not None:
+                context = get_project_context(
```

Comment:
How about using `load_context()` instead of `get_project_context()`?

Reply:
I gave a try, but it involves quite a bit of refactoring in the testing side, and slightly out of the PR scope. We can address this once we deprecate `get_project_context()` :)